src/api/types/playbooks.ts

```diff
@@ -58,7 +66,22 @@ export type PlaybookSpec = {
   icon?: string;
   created_by?: User;
   category?: string;
-  spec: any;
+  spec: {
+    actions?: Record<string, any>[];
+    configs?: {
+      type?: string;
```

Comment:
We also need tags in here

https://github.com/flanksource/mission-control/blob/c1b6368a9e1ebd010c8ab32965bfdf6320e74f0c/api/v1/playbook_types.go#L18-L23